src/leader.py

```diff
@@ -31,6 +32,16 @@ class Leader:
 
     STRAIGHT_ANGLE = 90
     """The angle that the hardware associates as straight."""
+    DEMO_SPEED = 40
+    """The speed the leader vehicle drives at."""
+    DEMO_STRAIGHT_TIME = 10
+    """The time the vehicle drives forward."""
+    DEMO_TURN_TIME = 5
+    """The time the vehicle turns"""
+    DEMO_RIGHT_TURN = 120
```

Comment:
Why can't we demo at max turns? Then we can just use the two functions we already have.